src/bundle/Resources/public/scss/_main-menu.scss

```
@@ -183,6 +160,10 @@
 
     &__item-icon-column {
         padding: calculateRem(10px);
+
+        .ibexa-icon {
+            fill: $ibexa-color-white;
+        }
```

Nitpick: I don't know if you like this approach, but sometimes I add `fill` to the parent and the icon just inherits it.
```suggestion
        fill: $ibexa-color-white;
```

---

I prefer to set the style to a specific node

---

Why not use `ibexa-icon--light`?